src/utils/nonce.ts

```diff
@@ -1,11 +1,16 @@
-import crypto from 'crypto';
+import {crypto} from '../runtime/crypto';
 
 export function nonce(): string {
   const length = 15;
-  const bytes = crypto.randomBytes(length);
+
+  // eslint-disable-next-line no-warning-comments
+  // TODO Remove the randomBytes call when dropping Node 14 support
+  const bytes = crypto.getRandomValues
+    ? crypto.getRandomValues(new Uint8Array(length))
+    : (crypto as any).randomBytes(length);
```

Comment:
Again, Node 14 doesn't support the more uniform way of doing this, so we need to default back to the original behaviour.